client/src/ui/components/Button.tsx

```diff
@@ -17,6 +18,7 @@ export const Button = ({
   iconEnd,
   iconStart,
   isLoading,
+  isDisabledBelowDropdown,
```

Comment:
@pradel I added this prop because when disabled button was with `opacity-50`, it was appearing in front of dropdown menu. 

For all the other disabled cases this opacity way works fine, but for the case below dropdown it didn't not so therefore the hacky way. 

Do you see any better way this could be fixed?